internal/service/ipam/model_ipamsvc_subnet.go

```diff
@@ -492,7 +506,7 @@ func (m *IpamsvcSubnetModel) Flatten(ctx context.Context, from *ipam.IpamsvcSubn
 	m.DdnsUpdateOnRenew = types.BoolPointerValue(from.DdnsUpdateOnRenew)
 	m.DdnsUseConflictResolution = types.BoolPointerValue(from.DdnsUseConflictResolution)
 	m.DhcpConfig = FlattenIpamsvcDHCPConfig(ctx, from.DhcpConfig, diags)
-	m.DhcpHost = flex.FlattenStringPointer(from.DhcpHost)
+	m.DhcpHost = flex.FlattenStringPointerWithNilAsEmpty(from.DhcpHost)
```

Comment:
Why did you have to do this? Because the API returns null?

Reply:
Yes.